md/taxonomy.md

```diff
@@ -257,5 +258,4 @@
   * [Map authorization rules](custom-authorization-rule-mapping.md)
   * [Use the rich textarea widget in a mail template](rta-mail-template.md)
   * [Manage dates in BDM and User Interfaces](dates-management-tutorial.md)
-
-
+  * [Install Tomcat with Bonita as a service in Windows](bonita-as-windows-service.md)
```

Comment:
I have just discussed with @NathalieC about the taxonomy
We think a better place for this is in the `Platform installation examples` part of the `Installation` section.
How-tos are for Bonita usage
❓ what do you think about this change?

Reply:
Sorry for the late answer, I think it's ok in the Platform installation examples. I'm just lacking time to complete...